src/mumble/ALSAAudio.cpp

```diff
@@ -382,8 +382,11 @@ void ALSAAudioInput::run() {
 #endif
 		readblapp = snd_pcm_readi(capture_handle, inbuff, static_cast<int>(wantPeriod));
 		if (readblapp == -ESTRPIPE) {
-			// suspend event - what to do?
-			qWarning("ALSAAudioInput: %s", snd_strerror(static_cast<int>(readblapp)));
+			qWarning("ALSAAudioInput: PCM suspended, trying to resume");
+			while (snd_pcm_resume(capture_handle) == -EAGAIN && bRunning)
+				this->msleep(1000);
+			if ((err = snd_pcm_prepare(capture_handle)) < 0)
```

Comment:
Should this be gated with "bRunning &&" to avoid calling snd_pcm_prepare when the device is not ready for it?

I don't know if bad things can happen by calling snd_pcm_prepare() if we broke out of the loop above with bRunning == false, without having called snd_pcm_resume() succesfully first?